### PR TITLE
RIA-3332:Home office validation flow with F&P changes test

### PR DESCRIPTION
**Pull request by divisathyan**
RIA-3332:Home office validation flow with F&P changes
No code change but to test functionality of Home Office after F&P changes were merged to master

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3332

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by jkumar-moj — APPROVED** [review]
👍

**Review by jkumar-moj — DISMISSED** [review]
👍

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java [hunk 1]
```diff
@@ -48,7 +48,8 @@ public boolean canHandle(
         return isHomeOfficeIntegrationEnabled
             && callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
             && (callback.getEvent() == Event.SUBMIT_APPEAL
-            || callback.getEvent() == Event.PAY_AND_SUBMIT_APPEAL)
+            || callback.getEvent() == Event.PAY_AND_SUBMIT_APPEAL
+            || callback.getEvent() == Event.MARK_APPEAL_PAID)
             && (callback.getCaseDetails().getState() == APPEAL_SUBMITTED
```
Comment (jkumar-moj) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java:53]: Hi @divisathyan, Could you please take out these state checks as well to be in consistent with HO-integration-api?

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidatePreparer.java [hunk 2]
```diff
@@ -33,7 +33,8 @@ public boolean canHandle(
 
         return callbackStage == PreSubmitCallbackStage.ABOUT_TO_START
             && (callback.getEvent() == Event.SUBMIT_APPEAL
-            || callback.getEvent() == Event.PAY_AND_SUBMIT_APPEAL)
+            || callback.getEvent() == Event.PAY_AND_SUBMIT_APPEAL
+            || callback.getEvent() == Event.MARK_APPEAL_PAID)
             && (callback.getCaseDetails().getState() == APPEAL_SUBMITTED
```
Comment (jkumar-moj) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidatePreparer.java:38]: same as above.

**Review by jkumar-moj — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java (hunk 1 above)
Comment (divisathyan) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java]: Removed state check and squashed all commits to one

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidatePreparer.java (hunk 2 above)
Comment (divisathyan) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidatePreparer.java]: Resolved

**Review by jkumar-moj — DISMISSED** [review]
lgtm 👍

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**